example-transformers/return-multiple-node/transformer.ts

```diff
@@ -0,0 +1,22 @@
+import * as ts from 'typescript';
+
+const transformer: ts.TransformerFactory<ts.SourceFile> = context => {
+  return sourceFile => {
+    const visitor = (node: ts.Node): ts.VisitResult<ts.Node> => {
+      // If it is a expression statement,
+      if (ts.isExpressionStatement(node)) {
+        // Clone it
+        const newNode = ts.getMutableClone(node);
```

Comment:
watch out for this - `ts-loader` doesn't play nicely (theres a section at the bottom of the handbook that mentions this). only found this out yesterday hehe

could you re-do this with either updateExpressionStatement or just returning the node twice?

[node, node]